### PR TITLE
Newsletter 107

### PR DESCRIPTION
**Pull request by Mohamed-amin**

### CODE REVIEW

File: source/_posts/2015-09-10-newsletter-number-107.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Newsletter #107"
+date: 2015-09-10 15:10
+comments: true
+author: Mohamed Amin
+categories: [coding, css, devops, git, haskell, html, osx, swift]
+---
```
Comment (Mohamed-amin) [inline source/_posts/2015-09-10-newsletter-number-107.markdown]: @cirpo @odino

File: source/_posts/2015-09-10-newsletter-number-107.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+> Hi there!
+>
+> Tech Thursday has come again!, We had a big event last week! our Nerdy managers went to berlin last week to participate in the Rocket internet Summit
```
Comment (p16) [inline source/_posts/2015-09-10-newsletter-number-107.markdown]: extra `,`

**Comment by p16** [comment]
:+1:

File: source/_posts/2015-09-10-newsletter-number-107.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+comments: true
+author: Mohamed Amin
+categories: [coding, css, devops, git, haskell, html, osx, swift]
+---
+
+> Hi there!
+>
+> Tech Thursday has come again! We had a big event last week!
+>
+> Our nerdy "managers" went to berlin last week to participate in the Rocket internet Summit 2015
```
Comment (razanbilwani) [inline source/_posts/2015-09-10-newsletter-number-107.markdown]: Berlin with a capital 'B'

Wasn't it the Rocket Internet 'Tech' Summit

File: source/_posts/2015-09-10-newsletter-number-107.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+> Links? Here they are!
+>
+
+* ["without google maps we'd be lost" - syrian refugee explains how he has been using apps bbc.co.uk/panorama ](https://www.facebook.com/bbcnews/videos/10153081461092217/)
+
+* [better commit messages with a .gitmessage template](https://robots.thoughtbot.com/better-commit-messages-with-a-gitmessage-template)
+
+* [devops bookmarks](http://www.devopsbookmarks.com/)
+<!-- more -->
+* [don’t fear the internet basico html & css for non-web designers](http://www.dontfeartheinternet.com/)
```
Comment (razanbilwani) [inline source/_posts/2015-09-10-newsletter-number-107.markdown]: basico?

**Pull request merged**